.make.versions

```diff
@@ -1,29 +1,31 @@
 ################################################################################
 # Here we attempt to capture/define all the version numbers used across the 
-# repository in Makefile format.
+# repository in Makefile format.  These are generally considered the version
+# numbers to be published on the next release/publishing of artifacts.
 ################################################################################
 
 # Data prep lab wheel version
-DPK_LIB_VERSION=0.1.1
-DPK_LIB_KFP_VERSION=0.1.1
+DPK_LIB_VERSION=0.2.0
+DPK_LIB_KFP_VERSION=0.2.0
```

Comment:
Considering that ingest to parquet adds a few changes to the libraries, do you want to postpone PyPI creation? or do this as a minor update?

Reply:
we can postpone publishing, probably until we're publishing transforms too

Reply:
That can be a bit tooo long